financial-connections/src/test/resources/robolectric.properties

```diff
@@ -0,0 +1,2 @@
+sdk=30
+
```

Comment:
Curious, why was this needed for Financial Connections?

Reply:
Roboelectric doesn't like compileSdkVersion 33, I think all modules using Roboelectric will need this as a workaround